src/features/defi/providers/fox-eth-lp/components/FoxEthLpManager/Deposit/components/Status.tsx

```diff
@@ -0,0 +1,214 @@
+import { useWallet } from 'hooks/useWallet/useWallet'
+import { bnOrZero } from 'lib/bignumber/bignumber'
+import { useFoxEthLpBalance } from 'pages/Defi/hooks/useFoxEthLpBalance'
+import {
+  selectAssetById,
+  selectFirstAccountSpecifierByChainId,
+  selectMarketDataById,
+  selectTxById,
+} from 'state/slices/selectors'
+import { serializeTxIndex } from 'state/slices/txHistorySlice/utils'
+import { useAppSelector } from 'state/store'
+
+import { FoxEthLpDepositActionType } from '../DepositCommon'
+import { DepositContext } from '../DepositContext'
+
+export const Status = () => {
+  const [userAddress, setUserAddres] = useState<string | null>(null)
+  const translate = useTranslate()
+  const { state, dispatch } = useContext(DepositContext)
+  const { query, history: browserHistory } = useBrowserRouter<DefiQueryParams, DefiParams>()
+  const { chainId } = query
+  const { opportunity } = useFoxEthLpBalance()
+  const {
+    state: { wallet },
+  } = useWallet()
+
+  const feeAssetId = toAssetId({
+    chainId,
+    assetNamespace: 'slip44',
+    assetReference: ASSET_REFERENCE.Ethereum,
+  })
+  const lpAsset = useAppSelector(state => selectAssetById(state, opportunity.assetId))
+  const foxAsset = useAppSelector(state => selectAssetById(state, foxAssetId))
+  const ethAsset = useAppSelector(state => selectAssetById(state, ethAssetId))
+
+  const feeMarketData = useAppSelector(state => selectMarketDataById(state, feeAssetId))
+
+  const accountSpecifier = useAppSelector(state =>
+    selectFirstAccountSpecifierByChainId(state, chainId),
+  )
+
+  useEffect(() => {
+    const chainAdapterManager = getChainAdapterManager()
+    const adapter = chainAdapterManager.get(ethAsset.chainId) as ChainAdapter<KnownChainIds>
+    if (wallet && adapter) {
+      ;(async () => {
+        const address = await adapter.getAddress({ wallet })
+        setUserAddres(address)
+      })()
+    }
+  }, [ethAsset.chainId, wallet])
+
+  const serializedTxIndex = useMemo(() => {
+    if (!(state?.txid && userAddress)) return ''
+    return serializeTxIndex(accountSpecifier, state.txid, userAddress)
+  }, [state?.txid, userAddress, accountSpecifier])
+  const confirmedTransaction = useAppSelector(gs => selectTxById(gs, serializedTxIndex))
+
+  useEffect(() => {
+    if (confirmedTransaction && confirmedTransaction.status !== 'Pending' && dispatch) {
+      dispatch({
+        type: FoxEthLpDepositActionType.SET_DEPOSIT,
+        payload: {
+          txStatus: confirmedTransaction.status === 'Confirmed' ? 'success' : 'failed',
+          usedGasFee: confirmedTransaction.fee
+            ? bnOrZero(confirmedTransaction.fee.value).div(`1e${ethAsset.precision}`).toString()
+            : '0',
+        },
+      })
+    }
+  }, [confirmedTransaction, dispatch, ethAsset.precision])
+
+  const handleViewPosition = () => {
+    browserHistory.push('/defi')
+  }
+
+  const handleCancel = () => {
+    browserHistory.goBack()
+  }
```

Comment:
One less track trace
```suggestion
  const handleCancel = browserHistory.goBack
```